playbooks/install_docker.yml

```
@@ -0,0 +1,49 @@
+        state: present
+
+    - name: Add user to the docker group
+      user:
+        name: user_stage
+        groups: docker
+        append: yes
+
+    - name: Restart docker service
+      service:
+        name: docker
+        state: restarted
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)